docs/use_cases/model_applications/clouds/GridStat_fcstGFS_obsGOES_cloudTopPressAndTemp.py

```diff
@@ -0,0 +1,167 @@
+# indicating which output are from which GridStat instance.
+
+##############################################################################
+# METplus Configuration
+# ---------------------
+#
+# METplus first loads all of the configuration files found in parm/metplus_config, 
+# then it loads any configuration files passed to METplus via the command line, 
+# i.e. parm/use_cases/model_applications/clouds/GridStat_fcstGFS_obsGOES_cloudTopPressAndTemp.conf
+#
+# .. highlight:: bash
+# .. literalinclude:: ../../../../parm/use_cases/model_applications/clouds/GridStat_fcstGFS_obsGOES_cloudTopPressAndTemp.conf
+
+##############################################################################
+# MET Configuration
+# -----------------
+#
+# METplus sets environment variables based on user settings in the METplus
+# configuration file. See :ref:`How METplus controls MET config file settings<metplus-control-met>` for more details.
+#
+# **YOU SHOULD NOT SET ANY OF THESE ENVIRONMENT VARIABLES YOURSELF! THEY WILL BE OVERWRITTEN BY METPLUS WHEN IT CALLS THE MET TOOLS!**
+#
+# If there is a setting in the MET configuration file that is currently
+# not supported by METplus you’d like to control, please refer to:
+# :ref:`Overriding Unsupported MET config file settings<met-config-overrides>`
+#
+# .. dropdown:: Point2GridConfig_wrapped
+#
+#   .. literalinclude:: ../../../../parm/met_config/Point2GridConfig_wrapped
+#
+# .. dropdown:: GridStatConfig_wrapped
+#
+#   .. literalinclude:: ../../../../parm/met_config/GridStatConfig_wrapped
+
+##############################################################################
+# Python Embedding
+# ----------------
+#
+# This use case does not use Python embedding.
+
+##############################################################################
+# User Scripting
+# --------------
+#
+# This use case does not use additional scripts.
+
+##############################################################################
+# Running METplus
+# ---------------
+#
+# Pass the use case configuration file to the run_metplus.py script along 
+# with any user-specific system configuration files if desired::
+#
+#   run_metplus.py /path/to/METplus/parm/use_cases/model_applications/clouds/GridStat_fcstGFS_obsGOES_cloudTopPressAndTemp.conf /path/to/user_system.conf
+#
+# See :ref:`running-metplus` for more information.
+
+##############################################################################
+# Expected Output
+# ---------------
+#
+# A successful run will output the following both to the screen and to the logfile::
+#
+#   INFO: METplus has successfully finished running.
+#
+# Refer to the value set for **OUTPUT_BASE** to find where the output data was generated. 
+# Output for this use case will be found in 
+# {OUTPUT_BASE}/model_applications/clouds/GridStat_fcstGFS_obsGOES_cloudTopPressAndTemp 
+# and will contain the following files::
+#
+#  * grid_stat_GFS_to_ACHTF_F06_CloudHgts_060000L_20240307_060000V_ctc.txt
```

Comment:
These file names need to be updated: after my runof the use case, I'm presented with file templates of grid_stat_GFS_vs_<VAR_NAME>_060000L_20240307_060000V.